src/store/TableDataStore.js

```diff
@@ -209,6 +209,11 @@ export class TableDataStore {
     }
   }
 
+  getDataIgnoringPagination() {
```

Comment:
@AllenFang exportCSV could just hit this.store.getDataIgnoringPagination() instead of this.store.get() to get all the results.